js/index.js

```diff
@@ -0,0 +1,94 @@
+const btn = document.querySelector('#newbook');
+const bookForm = document.querySelector('#bookform');
+
+function Book() {
+  this.title = document.querySelector('#booktitle').value;
+  this.author = document.querySelector('#author').value;
+  this.read = document.querySelector('#read').checked;
+  this.pages = document.querySelector('#pages').value;
+}
+
+// get books from localstorage
+const getBooks = () => {
+  const lib = localStorage.getItem('library');
+  return lib ? JSON.parse(lib) : [];
+};
+
+// loop list of books
+const render = (books) => {
+  let booksList = '';
+  books.forEach((book, i) => {
+    const { title, author, read, pages } = book;
+    booksList += `
+      <tr>
+        <th scope="row">${i + 1}</th>
+        <td>${title}</td>
+        <td>by <i>${author}</i></td>
+        <td>${pages ? pages : 'unknown'}</td>
+        <td> ${read}</td>
```

Comment:
- Each Book displayed should have a button to toggle the Book `read` status, see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/toggle_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library) - in this case I should be able to change the status when the book is being displayed.